### ISSUE TITLE
Show indicator for additional power

### ISSUE THREAD

**Issue by pappalar** [jeremylarner/keyteki#152]
Eg: Staunch Knight.

Would be nice to have indicators for additional power on top of the cards when the state is active

**Comment by wingedyeti** [comment]
I agree. It's very hard to track opponents numbers when there is no indicator that they have been powered up. Pretty broken when you think you'll kill something but don't.
